From 662560bad9be29c3e1a2668806392d2ed9a743d6 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Mon, 20 Jul 2026 17:16:30 +0100
Subject: [PATCH 06259/15333] drm/i915/bw: Fix spelling mistake "threshod" ->
 "threshold"

There is a spelling mistake in a drm_dbg_kms message. Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Link: https://patch.msgid.link/20260720161630.326016-1-colin.i.king@gmail.com
Signed-off-by: Rodrigo Vivi <rodrigo.vivi@intel.com>
---
 drivers/gpu/drm/i915/display/intel_bw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/display/intel_bw.c b/drivers/gpu/drm/i915/display/intel_bw.c
index f4ae6db9f567..99fd84a5ac13 100644
--- a/drivers/gpu/drm/i915/display/intel_bw.c
+++ b/drivers/gpu/drm/i915/display/intel_bw.c
@@ -633,7 +633,7 @@ static void xe3_add_peakbw_threshold(struct intel_display *display)
 	for (int i = 0; i < ARRAY_SIZE(display->bw.max); i++)
 		display->bw.max[i].deratedbw[qgv_points] = PEAK_BW_THRESHOLD;
 
-	drm_dbg_kms(display->drm, "An extra QGV point %d added for Peak bw threshod of %d\n",
+	drm_dbg_kms(display->drm, "An extra QGV point %d added for Peak bw threshold of %d\n",
 		    qgv_points, PEAK_BW_THRESHOLD);
 }
 
-- 
2.55.0

